### PR TITLE
Update PyTorch-Lighting minor version

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Solve #1669 and update the minor version of PyTorch-Lightning to `v0.8.1`.

## Description of the changes

This main change is to rename the deprecated method and arguments of PyTorch-Lightning. As a result, we need to update minor version to `pytorch_lightning>=v0.8.1` because `limit_val_batches` has been introduced since the version.

### CODE REVIEW

**Comment by nzw0301** [comment]
I should fix the `setup.py` for python 3.5...

File: optuna/integration/pytorch_lightning.py [hunk 1]
```diff
@@ -49,10 +49,5 @@ def _process(self, trainer: Trainer, pl_module: LightningModule) -> None:
             message = "Trial was pruned at epoch {}.".format(epoch)
             raise optuna.TrialPruned(message)
 
-    # NOTE (crcrpar): This method is called <0.8.0
-    def on_epoch_end(self, trainer: Trainer, pl_module: LightningModule) -> None:
-        return self._process(trainer, pl_module)
-
-    # NOTE (crcrpar): This method is called >=0.8.0
     def on_validation_end(self, trainer: Trainer, pl_module: LightningModule) -> None:
```
Comment (crcrpar) [inline optuna/integration/pytorch_lightning.py:52]: Could you remove `_process` and write its logic inside this method?

**Review by crcrpar — APPROVED** [review]
nice work, thank you.

**Review by HideakiImamura — APPROVED** [review]
LGTM!

**Pull request merged**